x-pack/plugins/security_solution/common/endpoint/data_generators/endpoint_action_generator.ts

```diff
@@ -89,9 +89,16 @@ export class EndpointActionGenerator extends BaseDataGenerator {
           type: 'json',
           content: {
             code: 'ra_get-file_success_done',
-            path: '/some/path/bad_file.txt',
-            size: 1234,
             zip_size: 123,
+            contents: [
+              {
+                type: 'file',
+                path: '/some/path/bad_file.txt',
+                size: 1234,
+                file_name: 'bad_file.txt',
+                sha256: '9558c5cb39622e9b3653203e772b129d6c634e7dbd7af1b244352fc1d704601f',
+              },
+            ],
```

Comment:
Yeah. Initially we wanted to details of the actual file so that we could show the file name without having to parse out the `path` or have to call the file info api. That need evolved and we no longer need it, but we had already asked the Endpoint to include it. We asked that he include the `contents` array that he also sends to the file meta when initializing the upload.

We might not use it now, but could in the future.